Plugins/NativeCamera/Source/Apple/NativeCameraImpl.mm

```diff
@@ -123,6 +123,9 @@ - (id)init:(std::shared_ptr<Babylon::Plugins::Camera::Impl::ImplData>)implData;
             [m_implData->avCaptureSession commitConfiguration];
             [m_implData->avCaptureSession startRunning];
         });
+        
+        // This should capture an actual task and return it... but we'll deal with that later
```

Comment:
I intend to address the iOS implementation so that the task completion properly awaits the async work before I complete the PR.

Reply:
The latest iteration now runs asynchronously and returns a proper task to await.